clearwater-cassandra/usr/share/clearwater/bin/poll_cassandra.sh

```diff
@@ -35,8 +35,12 @@
 # as those licenses appear in the file LICENSE-OPENSSL.
 
 # This script polls a cassandra process and check whether it is healthy by checking
-# that the 9160 port is open at cassandra_hostname.
+# that the 9160 port is open at cassandra_hostname. This script should be
+# run in the signaling namespace if set
+
 . /etc/clearwater/config
+. /usr/share/clearwater/bin/run-in-signaling-namespace
+
 [ ! -z "$cassandra_hostname" ] || cassandra_hostname="127.0.0.1"
-/usr/share/clearwater/bin/poll-tcp 9160 $cassandra_hostname
+$namespace_prefix /usr/share/clearwater/bin/poll-tcp 9160 $cassandra_hostname
```

Comment:
Why not just `/usr/share/clearwater/bin/run-in-signaling-namespace /usr/share/clearwater/bin/poll-tcp 9160 $cassandra_hostname`?